pkg/pgmodel/cache/series_cache.go

```diff
@@ -132,18 +135,28 @@ func (t *SeriesCacheImpl) loadSeries(str string) (l *model.Series) {
 // Try to set a series as the canonical Series for a given string
 // representation, returning the canonical version (which can be different in
 // the even of multiple goroutines setting labels concurrently).
-// This function should not be called directly, use labelProtosToLabels() or
-// LabelsFromSlice() instead.
+// This function should not be called directly, use GetSeriesFromProtos() or
```

Comment:
Maybe better just remove this comment line? I can't make any sense out of it.

Reply:
I _guess_ that the comment is saying something like "this is not the public api of the cache", which should be kind of obvious given the fact that the function name is lowercased (hence private).